src/layouts/ZclLayout.vue

```diff
@@ -144,9 +144,9 @@ limitations under the License.
                     clickable
                     v-close-popup
                     @click="
-                    generationButtonText = file.category
-                    getGeneratedFile(file.category)
-                  "
+                      generationButtonText = file.category
+                      getGeneratedFile(file.category)
+                    "
```

Comment:
See if you can get rid of these additional diffs so as to maintain the git history with a actual changes.

Reply:
I tried getting rid of those but when I committed and pushed there were "no changes to show". I assume it has to do with our formatter?